platform/core/src/services/ViewerToolsetService/ViewerToolset.js

```diff
@@ -4,16 +4,16 @@ const publicAPI = {
   name,
   getState: _getState,
   setIsReferenceLinesEnabled: _setIsReferenceLinesEnabled,
-  setIsCrosshairsEnabled: _setIsCrosshairsEnabled,
```

Comment:
Why is setIsCrosshairsEnabled removed now? Should it have both Crosshairs and seriesLinking?